Formula/avgtime.rb

```diff
@@ -0,0 +1,17 @@
+class Avgtime < Formula
+  desc "Like unix 'time' command, but with repetitions, median, std deviation"
+  homepage "https://github.com/jmcabo/avgtime"
+  url "https://github.com/jmcabo/avgtime.git", :revision => "ffdf20049db7a6aa6074354d95265869c6aad65b"
```

Comment:
the `url` and `version` aren't tagged releases as the maintainer is still pending cutting a new release. Will update this in future if that changes.